releasenotes/notes/add_ControlledGate_class-010c5c90f8f95e78.yaml

```diff
@@ -0,0 +1,27 @@
+---
+features:
+  - |
+    Add ``ControlledGate`` class for representing controlled
+    gates. Controlled gate instances are created with the
+    ``q_if(n)`` method of ``Gate`` objects where ``n`` represents
```

Comment:
if #3455 is accepted this has to be updated to `control`.. same for a few lines further down

Reply:
I wrote it for what is currently in but can change it for what's expected to be in.